src/praisonai-agents/praisonaiagents/agent/chat_mixin.py

```
@@ -234,7 +234,10 @@ def _supports_native_structured_output(self):
         try:
             from ..llm.model_capabilities import supports_structured_outputs
             return supports_structured_outputs(self.llm)
-        except Exception:
+        except ImportError:
+            return False  # Module genuinely not available — acceptable
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)